extras/rpc-spec.txt

```diff
@@ -676,10 +676,11 @@
 
    Response arguments:
 
-   string      | value type & description
-   ------------+----------------------------------------------------------
-   "path"      | string  same as the Request argument
-   "size-bytes"| number  the size, in bytes, of the free space in that directory
+   string       | value type & description
+   -------------+----------------------------------------------------------
+   "path"       | string  same as the Request argument
+   "size-bytes" | number  the size, in bytes, of the free space in that directory
+   "total_size" | number  the total size, in bytes, that exists in that directory
```

Comment:
1. Need to bump RPC version to 18 and annotate the `5.0.  Protocol Versions` section

2. Also, might rephrase the description a little? "total size that exists in that directory" is ambiguous, does it mean the how much data exists in that directory, or the directory's total capacity?